libFMS_mod/libFMS.F90

```diff
@@ -0,0 +1,471 @@
+                     mpp_update_domains, mpp_check_field, mpp_start_update_domains, &
+                     mpp_complete_update_domains, mpp_create_group_update, &
+                     mpp_do_group_update, mpp_start_group_update, &
+                     mpp_complete_group_update, mpp_reset_group_update_field, &
+                     mpp_update_nest_fine, mpp_update_nest_coarse, mpp_get_boundary, &
+                     mpp_update_domains_ad, mpp_get_boundary_ad, mpp_pass_SG_to_UG, &
+                     mpp_pass_UG_to_SG, mpp_define_layout, mpp_define_domains, &
+                     mpp_modify_domain, mpp_define_mosaic, mpp_define_mosaic_pelist, &
+                     mpp_define_null_domain, mpp_mosaic_defined, &
+                     mpp_define_io_domain, mpp_deallocate_domain, &
+                     mpp_compute_extent, mpp_compute_block_extent, &
+                     mpp_define_unstruct_domain, domainUG, mpp_get_UG_io_domain, &
+                     mpp_get_UG_domain_npes, mpp_get_UG_compute_domain, &
+                     mpp_get_UG_domain_tile_id, mpp_get_UG_domain_pelist, &
+                     mpp_get_ug_domain_grid_index, mpp_get_UG_domain_ntiles, &
+                     mpp_get_UG_global_domain, mpp_global_field_ug, &
+                     mpp_get_ug_domain_tile_list, mpp_get_UG_compute_domains, &
+                     mpp_define_null_UG_domain, NULL_DOMAINUG, mpp_get_UG_domains_index, &
+                     mpp_get_UG_SG_domain, mpp_get_UG_domain_tile_pe_inf, &
+                     mpp_define_nest_domains, mpp_get_C2F_index, mpp_get_F2C_index, &
+                     mpp_get_nest_coarse_domain, mpp_get_nest_fine_domain, &
+                     mpp_is_nest_coarse, mpp_is_nest_fine, &
+                     mpp_get_nest_pelist, mpp_get_nest_npes, &
+                     mpp_get_nest_fine_pelist, mpp_get_nest_fine_npes, &
+                     mpp_domain_UG_is_tile_root_pe, mpp_deallocate_domainUG, &
+                     mpp_get_io_domain_UG_layout, NULL_DOMAIN1D, NULL_DOMAIN2D
+
+  !> platform
+  use platform_mod, only: r8_kind, r4_kind, i8_kind, i4_kind, c8_kind, c4_kind, &
+                          l8_kind, l4_kind, i2_kind, ptr_kind
+
+  !> random_numbers
+  use mersennetwister_mod, only: randomNumberSequence, new_RandomNumberSequence, &
+                                finalize_RandomNumberSequence, getRandomInt, &
+                                getRandomPositiveInt, getRandomReal
+  use random_numbers_mod, only: randomNumberStream, initializeRandomNumberStream, &
+                                getRandomNumbers, constructSeed
+
+  !> sat_vapor_pres
+  use sat_vapor_pres_k_mod, only: sat_vapor_pres_init_k, lookup_es_k, lookup_des_k, &
+                                lookup_es_des_k, lookup_es2_k, lookup_des2_k, &
+                                lookup_es2_des2_k, lookup_es3_k, lookup_des3_k, &
+                                lookup_es3_des3_k, compute_qs_k, compute_mrs_k
```

Comment:
These should only be used by sat_vapor_pres.F90 and don't need to be made public.

Reply:
Removed with 056a29f